celest/schedule/schedule.py

```diff
@@ -1,6 +1,5 @@
-
-
-from celest.satellite.satellite import Satellite
+from celest.coordinates import Attitude
```

Comment:
Turn into full import statement.

```suggestion
from celest.coordinates.frames.attitude import Attitude
```